### ISSUE TITLE
`duplicate-names-in-diff-context` should not flag test attributes as duplicates

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#636]
This is my code:

```
[] > foo
  [] +> works-well
    42 > i
  [] +> works-very-well
    44 > i
```

I get a complaint from `duplicate-names-in-diff-context` because of `i` being used in two different contexts. I believe, we should not complain if these contexts are test attributes. They are like different programs. We should allow them having similar names inside.
